tests/phpunit/tests/general/feedLinksExtra.php

```diff
@@ -0,0 +1,468 @@
+<?php
+/**
+ * Test feed_links_extra().
+ *
+ * @ticket 54713
+ *
+ * @group general
+ * @group template
+ *
+ * @covers ::feed_links_extra
+ */
+class Tests_General_FeedLinksExtra extends WP_UnitTestCase {
+	protected static $author;
+	protected static $category;
+	protected static $tag;
+	protected static $tax;
+	protected static $post_type;
+	protected static $post_no_comment;
+	protected static $post_with_comment;
+	protected static $post_with_cpt;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		/** Author */
+		self::$author = $factory->user->create(
+			array(
+				'user_login' => 'author_feed_links_extra',
+				'role'       => 'administrator',
+			)
+		);
+
+		/** Category */
+		self::$category = $factory->category->create(
+			array( 'name' => 'cat_feed_links_extra' )
+		);
+
+		/** Tag */
+		self::$tag = $factory->tag->create(
+			array( 'name' => 'tag_feed_links_extra' )
+		);
+
+		/** Taxonomy */
+		self::$tax = 'tax_feed_links_extra';
+
+		/** Post type */
+		self::$post_type = 'cpt_feed_links_extra';
+
+		/** Posts */
+		self::$post_no_comment = $factory->post->create(
+			array( 'post_title' => 'Post with no comments' )
+		);
+
+		self::$post_with_comment = $factory->post->create(
+			array( 'post_title' => 'Post with a comment' )
+		);
+
+		$factory->comment->create(
+			array(
+				'comment_author'  => self::$author,
+				'comment_post_ID' => self::$post_with_comment,
+			)
+		);
+	}
+
+	public function set_up() {
+		parent::set_up();
+
+		register_taxonomy(
+			self::$tax,
+			self::$post_type,
+			array(
+				'labels' => array(
+					'name'          => 'Taxonomy Terms',
+					'singular_name' => 'Taxonomy Term',
+				),
+			)
+		);
+
+		register_post_type(
+			self::$post_type,
+			array(
+				'public'      => true,
+				'has_archive' => true,
+				'taxonomies'  => array( self::$tax ),
+				'labels'      => array( 'name' => 'CPT for feed_links_extra()' ),
+			)
+		);
+
+		self::$post_with_cpt = $this->factory->post->create(
+			array(
+				'post_title' => 'Post with a custom post type',
+				'post_type'  => self::$post_type,
+			)
+		);
+
+		wp_set_object_terms( self::$post_with_cpt, 'tax_term', self::$tax );
+	}
+
+	public function tear_down() {
+		parent::tear_down();
+
+		_unregister_taxonomy( self::$tax );
+		_unregister_post_type( self::$post_type );
+	}
+
```

Comment:
```suggestion
```

Post types and taxonomies are automatically reset on tear_down. [ref](https://github.com/WordPress/wordpress-develop/blob/8a4ffe388300701a0205d7d3ca9ea8fa4e861031/tests/phpunit/includes/abstract-testcase.php#L268-L296)